README.md

```diff
@@ -1,15 +1,188 @@
-Welcome to your new dbt project!
+# Avo Audit
 
-### Using the starter project
+Avo audit helps you detect issues in your raw analytics event tables by querying a sample of the raw events and comparing them against each other.
+This allows you to find discrepancies between events and properties that should look the same, but don’t.
 
-Try running the following commands:
-- dbt run
-- dbt test
+## Issue types
 
+Issues available in the alpha version:  
 
-### Resources:
-- Learn more about dbt [in the docs](https://docs.getdbt.com/docs/introduction)
-- Check out [Discourse](https://discourse.getdbt.com/) for commonly asked questions and answers
-- Join the [chat](http://slack.getdbt.com/) on Slack for live discussions and support
-- Find [dbt events](https://events.getdbt.com) near you
-- Check out [the blog](https://blog.getdbt.com/) for the latest news on dbt's development and best practices
+**Property sometimes missing** – detects absence of event properties across tracking calls.  
+>`Property onboarding status is not sent with 30% of the App Opened event.`  
+
+**Property type mismatch** – detects inconsistent event property types across tracking calls.  
+>`Inconsistent type of property onboarding status on event App Opened: int (84%), string (16%).`
+
+### Issue types on the roadmap (not prioritized list):
+* Event missing on some platforms
+* Property missing on some platforms
+* Event volume change significant between versions
+* Event volume significantly different between platforms
+* Inconsistent event name casing
+* Inconsistent property name casing
+* Global property type mismatch
+* Similar event names
+* Similar property names within event
+* Unexpected type based on property name
+* Global similar property names
+* Missing property based on property group pattern
+
+
+## How do I get the most out of this package? 
+We recommend implementing at least the main 3 macros, and chain them together. That way you will generate a SQL table listing the issues found in your project, ordered by the priority listed above.
+
+Steps:
+1. Convert raw events into it’s signature type with **parse_signature**
+2. Group events by identical signatures and compare events with different signatures Per event with **compare_events**
+3. Generate issues table by using the **report_issues** macro.
```

Comment:
Should we link to the macros here somehow? Either some sort of a reference doc, or directly to the code (if it exists)